.env.example

```diff
@@ -11,6 +11,7 @@ VUE_APP_I18N_LOCALE=en
 VUE_APP_I18N_FALLBACK_LOCALE=en
 VUE_APP_UPDATE_INFO_INTERVAL=60000
 VUE_APP_UPDATE_CONTACTS_INTERVAL=0
+VUE_APP_RECORDING_DISABLE_PLAY_BUTTON=false
```

Comment:
Let's rename this env variable to `VUE_APP_WEBTRIT_RECORDING_PLAY_BUTTON_DISABLE` and put it after `VUE_APP_WEBTRIT_NAVIGATION_DRAWER_MINIMIZING`.